ParisBNF/abb/BNFabb147.xml

```diff
@@ -147,7 +148,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            <summary/>
 
                         <msItem xml:id="p_i1">
-                          <locus from="1a" to="122b"/>
+                          <locus from="1ra" to="66vb">1a-122b</locus>
```

Comment:
see above, all depends on whether i-ii become 1-2.
if yes then pp. 1-122 become ff. 3-63v
if not then pp. 1-122 are simply, divided by 2, ff. 1-61
whichever way 66 is wrong

check the formula I suggested for calculating page from fol. https://github.com/BetaMasaheft/Documentation/issues/1851